docs/sources/example-rules/dropbox-business/dropbox-business.yaml

```diff
@@ -31,6 +31,8 @@ endpoints:
           - "$.events[*].assets[*]['display_name', 'doc_title', 'folder_title', 'showcase_title']"
           - "$.events[*].assets[*].path['contextual']"
           - "$.events[*].assets[*].path.namespace_relative['relative_path']"
+    supportedHeaders:
+      - "Dropbox-API-Select-User"
```

Comment:
While testing this on dropbox, I've discovered that the connector is not working; something related with tokens. I'll fix it in a different PR.

Reply:
https://app.asana.com/0/1205303754976321/1205322459838014

Reply:
i don't really care; mark it `alpha` everywhere and we can return to it later

Reply:
Done in https://github.com/Worklytics/evalengin/pull/6000